CivicActions/pgov-cms
# PGOV-339: Add Search API/Facets and build Goals view

To test:

Pull code
Run composer install to get the new search api and facet modules
Run config import to install new modules 
Run cron(maybe?) or visit the search api admin page to index all the goals for the view
Start the nextjs app
You should now have a homepage view of goals, with topic filters and a search bar

The current interactions available:
To get around some planning for how to handle subsequent requests to the Drupal site, the app just builds with the 400 goals already loaded into React. When you choose a filter it will only show the goals with that topic. If you click multiple filters it will show any goals that have any of those topics. A search will currently just do a string match on the title and description. It will only return goals that have match and disable any filters that don't apply to the returned goals. 


There are some issues I noticed, like if you search 800 number, there are 3 results but only 2 of them have topics associated with them.
